src/Popper.jsx

```diff
@@ -139,13 +139,15 @@ class Popper extends Component {
       left,
       position,
     } = this.state.data.offsets.popper
+    const styles = this.state.data.styles
 
     return {
       position,
       top:        0,
       left:       0,
       transform:  `translate3d(${left}px, ${top}px, 0px)`,
       willChange: 'transform',
+      ...styles,
```

Comment:
You don't need any of the above definitions if you are going to use the `styles` provided by Popper.js, the code here can be simplified.